src/raygun.rum.js

```diff
@@ -734,6 +748,28 @@ var raygunRumFactory = function (window, $, Raygun) {
                 }
             }
         }
+        
+        function postSuccessCallback() {
+            self.postAttempts = 0;
```

Comment:
What happens here if

1. Request is started to server and it runs into a timeout issue
2. Another page is navigated to, causing another request to be triggered to the server and this one doesn't timeout (and happens before the first request has timed out so it hasn't been re-tried)

The second request being successful would reset the post attempts of the first request? I feel like these should be per request variables? Not entirely sure how you would pull that off, but I suspect some trickyness with closures could do it.

Reply:
That's a good point, I'll look into this